### PR TITLE
Align some naming convention with spring boot.

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
* Rename clientId -> client-id, clientSerect -> client-serect

* Rename azureservice -> azure.service

Signed-off-by: Pan Li <panli@microsoft.com>

## Issue Type

- New Feature
- Bug fixing

## Starter Names
  
  - active directory spring boot starter
  - documentdb spring boot starter
  - key vault spring boot starter
  - media services spring boot starter
  - service bus spring boot starter
  - storage spring boot starter

## Additional Information

### CODE REVIEW

**Comment by sophiaso** [comment]
refs #281

File: azure-spring-boot-samples/azure-active-directory-spring-boot-sample/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # By default, azure.activedirectory.environment property has value `global`,
 # supported value is global, cn. Please refer to the README for details.
 # azure.activedirectory.environment=global
-azure.activedirectory.clientId=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-azure.activedirectory.clientSecret=ABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHIJKLMNOPQR
+azure.activedirectory.client-id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+azure.activedirectory.client-secret=ABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHIJKLMNOPQR
```
Comment (yungez) [inline azure-spring-boot-samples/azure-active-directory-spring-boot-sample/src/main/resources/application.properties]: also update readme docs?

**Comment by yungez** [comment]
@snicoll

File: azure-spring-boot-samples/azure-active-directory-spring-boot-sample/README.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ The sample is composed of two layers: Angular JS client and Spring Boot RESTful
 #### Application.properties
 You need to have an registered app in your Azure AD tenant and create a security key.
 If your azure account follows format xxx@xxx.partner.onmschina.cn, configure property `azure.activedirectory.environment=cn` to use [Azure China](https://docs.microsoft.com/en-us/azure/china/china-welcome), the default value is `global`.
-Put Application ID and Key in `clientId` and `clientSecret` respectively e.g.
-`azure.activedirectory.clientId=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`
-`azure.activedirectory.clientSecret=ABCDEFGHIJKLMNOOPQRSTUVWXYZABCDEFGHIJKLMNOPQ`
+Put Application ID and Key in `client-id` and `client-secret` respectively e.g.
```
Comment (yungez) [inline azure-spring-boot-samples/azure-active-directory-spring-boot-sample/README.md]: here also, https://github.com/Microsoft/azure-spring-boot/blob/master/azure-spring-boot-starters/azure-active-directory-spring-boot-starter/README.md

File: azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/ServiceEndpointsProperties.java [hunk 3]
```diff
@@ -13,7 +13,7 @@
 import java.util.Map;
 
 @Configuration
-@ConfigurationProperties("azureservice")
+@ConfigurationProperties("azure.service")
```
Comment (yungez) [inline azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/ServiceEndpointsProperties.java]: seems there's no instruction in document about how to set this property?

File: azure-spring-boot-samples/azure-active-directory-spring-boot-sample/src/main/resources/application.properties (hunk 1 above)
Comment (Incarnation-p-lee) [reply azure-spring-boot-samples/azure-active-directory-spring-boot-sample/src/main/resources/application.properties]: Done.

File: azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/ServiceEndpointsProperties.java (hunk 3 above)
Comment (Incarnation-p-lee) [reply azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/ServiceEndpointsProperties.java]: Yes, I try to find that word in repo including README. But only below change appears. We may enhance this in short future.

File: azure-spring-boot/src/main/resources/serviceEndpoints.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-azureservice:
+azure.service:
```
Comment (snicoll) [inline azure-spring-boot/src/main/resources/serviceEndpoints.yml]: Could be multiline here.

```yml
azure:
  service:
    endpoints:
```

Comment (Incarnation-p-lee) [reply azure-spring-boot/src/main/resources/serviceEndpoints.yml]: Thanks and updated.

**Pull request merged**